src/ucp/core/ucp_worker.c

```diff
@@ -2524,28 +2527,39 @@ static void ucp_worker_destroy_eps(ucp_worker_h worker,
     }
 }
 
+static void ucp_worker_eps_cleanup(ucp_worker_h worker)
+{
+    /* Cleanup hash of discarded UCT EPs prior destroying all UCP EPs to
+     * destroy UCP EPs which were fully discarded */
```

Comment:
maybe just `Cleanup hash of discarded UCT EPs prior destroying all UCP EPs`

Reply:
better to have an explanation of why it is needed prior to destroying all EPs.